repos/system_upgrade/el9toel10/actors/sssd/sssdupdate/libraries/sssdupdate.py

```diff
@@ -0,0 +1,74 @@
+import os
+
+from leapp.exceptions import StopActorExecutionError
+
+def _process_knownhosts(line:str) -> str:
+    if 'sss_ssh_knownhostsproxy' in line:
+        # Update the line, leaving intact any # and any --domain/-d parameter
+        line = line.replace('ProxyCommand', 'KnownHostsCommand')
+        line = line.replace('sss_ssh_knownhostsproxy', 'sss_ssh_knownhosts')
+        line = line.replace('--port=', '')
+        line = line.replace('--port', '')
+        line = line.replace('-p=', '')
+        line = line.replace('-p', '')
+        line = line.replace('%p', '')
+        line = line.replace('%h', '%H')
```

Comment:
Just to be extra cautions. Can you think of some weird syntax/quoting that might be lead to problems when using these kinds of replacements?

Reply:
Sorry, I just noticed that in sssdupdate actor's tests, this is expected. However I am still not sure we should touch commented out configuration. @MichalHe wdyt?